iModelCore/ECPresentation/Source/Content/ContentItemBuilder.cpp

```diff
@@ -25,7 +25,9 @@ rapidjson::Document ContentValuesFormatter::GetFallbackPrimitiveValue(ECProperty
     rapidjson::Document json(allocator);
     ECValue v = ValueHelpers::GetECValueFromSqlValue(type, extendedType, value);
     Utf8String stringValue;
-    if (v.ConvertPrimitiveToString(stringValue))
+    if (v.IsNull())

```

Comment:
Currently we are making the assumption that ECValue will not be null, but i think it is worth checking. Theoretically it could happen, since `ValueHelpers::GetECValueFromSqlValue` can return a ECValue that is set to null.